dask_ml/model_selection/_incremental.py

```diff
@@ -256,7 +258,15 @@ def get_futures(partial_fit_calls):
 
     models = {k: client.submit(operator.getitem, v, 0) for k, v in models.items()}
     yield wait(models)
-    raise gen.Return(Results(info, models, history))
+    scores = yield client.gather(scores)
```

Comment:
@mrocklin is awaiting the scores here a bad idea?

Reply:
Seems ok to me